docs/pct/README.md

```diff
@@ -0,0 +1,291 @@
+# Puppet Content Templates
+
+* [Overview](#overview)
+* [Getting Started](#getting-started)
+  * [pct new](#pct-new)
+  * [Template Updates](#template-updates)
+  * [More Templates](#more-templates)
+* [Writing Your Own Templates](#writing-templates)
+  * [Dos and Don'ts](#dos-and-donts)
+  * [Structure](#structure)
+    * [pct-config.yml](#pct-configyml)
+  * [Templating Language](#templating-language)
+  * [Testing](#testing)
+  * [Sharing](#sharing)
+* [Overriding Template Defaults](#overriding-template-defaults)
+
+## Overview
+
+`Puppet Content Templates` (PCT) allow you to quickly create Puppet content from scaffolding templates built by Puppet and the Community.
```

Comment:
You shouldn't put a code block around the 'Puppet Content Templates' phrase unless you're going to do it for the entire document. In general we use code blocks for language keywords not the name of the language itself.

Reply:
I think we can use some of the language in the specification document to flesh out the intro here. We have the opportunity to describe the purpose as well as show why to use the product in the intro